x-pack/test/dataset_quality_api_integration/tests/data_streams/data_stream_settings.spec.ts

```diff
@@ -99,11 +103,11 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         expect(err.res.body.message.indexOf(expectedMessage)).to.greaterThan(-1);
       });
 
-      it('returns {} if matching data stream is not available', async () => {
+      it('returns only privileges if matching data stream is not available', async () => {
```

Comment:
Would this make sense?
If non-matching dataStream I would expect to get empty or maybe an error.
Am I missing something?

Reply:
[Duplicate](https://github.com/elastic/kibana/pull/189532#discussion_r1714912140)